hnn_core/cells_default.py

```diff
@@ -280,3 +280,34 @@ def pyramidal(cell_name, pos=(0, 0, 0), override_params=None, gid=None):
                 topology=topology,
                 sect_loc=sect_loc,
                 gid=gid)
+
+
+def _get_g_at_dist(x, gsoma, gdend, xkink):
```

Comment:
some tests should be added for this. I think you can just import `pyramidal_ca` inside the test then do: `pyramidal_ca.build()` and check that the conductances are indeed what they should be ... e.g., at `x > xkink` etc

Reply:
dumb neuroscience question: what's the motivation/rationale for distance-dependence of the calcium dynamics?

Reply:
I think Sarah cited some evidence in her thesis, though as I recall there's evidence for gradients in both directions... The present value set created the desired spiking responses in the dendrites.

Reply:
@jasmainak [597efef](https://github.com/jonescompneurolab/hnn-core/pull/333/commits/597efef7bfb13d1abb91a7c62759e537002b6498) has a test ensuring a distance dependent change in gbar. Let me know what you think!